codecs/resize/src/lib.rs

```diff
@@ -44,21 +47,24 @@ fn converter_funcs(with_space_conversion: bool) -> ((fn(u8) -> f32), (fn(f32) ->
 // false, the functions just return the channel value.
 fn alpha_multiplier_funcs(
     with_alpha_premultiplication: bool,
-) -> ((fn(f32, u8) -> u8), (fn(u8, u8) -> f32)) {
+) -> (fn(f32, f32) -> f32, fn(f32, f32) -> f32) {
     if with_alpha_premultiplication {
-        (
-            |v, a| (v * (a as f32) / 255.0) as u8,
-            |v, a| ((v as f32) * 255.0 / (a as f32)).clamp(0.0, 255.0),
-        )
+        (|v, a| v * a, |v, a| v / a)
     } else {
-        (|v, _a| v as u8, |v, _a| v as f32)
+        (|v, _a| v, |v, _a| v)
     }
 }
 
+fn vec_with_len<T: Clone>(len: usize, item: T) -> Vec<T> {
```

Comment:
This exists in stdlib as a standard macro.

Reply:
... and that macro would be? :D

Reply:
ah just saw below, lol. Great job, GitHub.